### PR TITLE
Argo CD Fed ETL Example

### PR DESCRIPTION
**Pull request by mclean0328**
Adding an example of how to deploy Kubecost using federated etl with Argo CD. 

Also reorganized the folder to have sub-folders for both the thanos deployment example and the federated etl deployment example.

### CODE REVIEW

File: argocd/federated-etl-argo/argocd-kubecost-primary.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+spec:
+  project: default
+  source:
+    chart: cost-analyzer
+    repoURL: https://kubecost.github.io/cost-analyzer/
+    targetRevision: 1.105.7
+    helm:
+      releaseName: kubecost
+      valueFiles:
+      # Be sure to set the relative path to the secondary values file kubecost-primary-federator-values.yaml
```
Comment (srpomeroy) [inline argocd/federated-etl-argo/argocd-kubecost-primary.yaml:15]: Nit: Should this be `primary`?
```suggestion
      # Be sure to set the relative path to the primary values file kubecost-primary-federator-values.yaml
```

File: argocd/federated-etl-argo/argocd-kubecost-primary.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+kind: Application
+metadata:
+  name: kubecost
+  namespace: argocd
+spec:
+  project: default
+  source:
+    chart: cost-analyzer
+    repoURL: https://kubecost.github.io/cost-analyzer/
+    targetRevision: 1.105.7
```
Comment (srpomeroy) [inline argocd/federated-etl-argo/argocd-kubecost-primary.yaml:11]: 1.105.7?

**Review by srpomeroy — COMMENTED** [review]
Two minor nits.

File: argocd/federated-etl-argo/argocd-kubecost-primary.yaml (hunk 2 above)
Comment (mclean0328) [reply argocd/federated-etl-argo/argocd-kubecost-primary.yaml:11]: oh man, this should definitely be 1.105.1.  Thank you!

**Review by srpomeroy — APPROVED** [review]
LGTM!

**Pull request merged**